meteor/client/lib/reactiveData/reactiveDataHelper.ts

```diff
@@ -66,12 +65,11 @@ export function memoizedIsolatedAutorun<T extends (...args: any) => any>(
 
 	let result: ReturnType<T>
 	const fId = hashFncAndParams(functionName, params)
-	const parentComputation = Tracker.currentComputation
 	if (isolatedAutorunsMem[fId] === undefined) {
 		const dep = new Tracker.Dependency()
 		dep.depend()
-		const computation = Tracker.nonreactive(() => {
-			const computation = Tracker.autorun(() => {
+		const computationNonreactive = Tracker.nonreactive(() => {
+			const computationAutorun = Tracker.autorun(() => {
```

Comment:
So this is actually something I have a problem with. This `computationAutorun` and `computationNonreactive` are exactly the same thing. `Tracker.nonreactive` simply passes through, whatever is returned by the inner function. When written like this, it invites the thinking that these are two different computations and may cause someone to, for example, register two `onStop` or `onInvalidate` handlers. Line 94 becomes especially confusing.

Reply:
Good point! I'll revert that change and mark it as wontfix in SonarCloud.